test/integration/strong_mode_test.dart

```diff
@@ -0,0 +1,89 @@
+@TestOn('vm')
+library over_react.strong_mode;
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:dart_dev/util.dart' show copyDirectory;
+import 'package:test/test.dart';
+
+const String strongModeProject = 'test_fixtures/strong_mode';
+
+/// Runs the dart analzyer task for the given project.
+Future<ProcessResult> analyzeProject(String projectPath) async {
+  var files = <String>[]
+    ..addAll(new Directory(projectPath).listSync()
+        .where((e) => FileSystemEntity.isFileSync(e.path) && e.path.endsWith('.dart'))
+        .map((e) => e.path)
+    );
+
+  var args = ['--strong']
+    ..addAll(files);
+
+  return await Process.run('dartanalyzer', args);
```

Comment:
Hm, interesting.

I'd try using the project path as the working directory for the command. If that doesn't work, that's fine, but we should have a comment here as to why we're listing out all of the files.